Dockerfile

```diff
@@ -70,6 +99,10 @@ RUN cabal update && \
     ihaskell install && \
     rm -fr $(echo ~/.cabal/bin/* | grep -iv ihaskell) ~/.cabal/packages ~/.cabal/share/doc ~/.cabal/setup-exe-cache ~/.cabal/logs
 
+# Spark kernel spec
+RUN mkdir -p $HOME/.ipython/kernels/spark
```

Comment:
Good catch. That was a naming mixup on my part. I've forced pushed and corrected the directory to `scala` along with the comment.